products/motorola.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Motorola Mobility
+category: device
+tags: mobile-phone
+iconSlug: motorola
+permalink: /motorola
+alternate_urls:
+-   /motorolamobility
+-   /motorola-mobility
+releasePolicyLink: https://en-us.support.motorola.com/app/software-security-update
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: false
```

Comment:
```suggestion
```

`false` is already the default value (see https://github.com/endoflife-date/endoflife.date/blob/master/_config.yml).